### PR TITLE
lib/pq version instead of master

### PR DESCRIPTION
**Pull request by urakozz**
Updated lib/pq dependency
removed vendors from git

### CODE REVIEW

**Comment by urakozz** [comment]
@kevinburke could you probably check and merge this PR?

**Comment by kevinburke** [comment]
Hey, sorry! Looking now

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /*.out
 /*.test
+/.idea
+/vendor
```
Comment (kevinburke) [inline .gitignore:4]: I'd prefer to keep the vendor directory checked in.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 /*.out
 /*.test
+/.idea
```
Comment (kevinburke) [inline .gitignore:3]: Use .git/info/exclude for this, it's not a project artifact, it's an artifact of your build environment. https://stackoverflow.com/q/22906851

**Comment by kevinburke** [comment]
Merged as 301feef841ad2788c2bbae3f7e959347f7dec1da, thanks!

**Pull request closed without merging**